akka-docs/src/main/paradox/common/may-change.md

```diff
@@ -31,5 +31,5 @@ These are the current complete modules marked as **may change**:
 * @ref:[Multi Node Testing](../multi-node-testing.md)
 * @ref:[Reliable Delivery](../typed/reliable-delivery.md)
 * @ref:[Sharded Daemon Process](../typed/cluster-sharded-daemon-process.md)
-
+* DurableStateBehavior
```

Comment:
@debasishg this should be replaced with link to the doc page when that is ready

Reply:
ok ..